src/switchroot/ostree-prepare-root.c

```diff
@@ -77,6 +78,7 @@
 #include "otcore.h"
 
 #define SYSROOT_KEY "sysroot"
+#define INVISIBLE_KEY "invisible"
```

Comment:
Minor bikeshed; since this overlaps/replaces `sysroot.readonly` how about we have just
`sysroot.visibility = readonly | hidden` or so?

We'd continue to parse `sysroot.readonly = true` and just turn it into `sysroot.visibility = readonly`.

In general "enums" (as expressed as strings) here feel more extensible than bools.

Reply:
Good idea.

Reply:
I also added an alias `sysroot.protect`. IMO `sysroot.protect=invisible` looks better.